composer.json

```diff
@@ -20,11 +20,11 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^8.1",
```

Comment:
How's that change related to the new feature?

Reply:
Collections 2.1 only accepts php ^8.1

Reply:
This is why you cannot drop Collections 1. Sorry, this change is a show-stopper.

Reply:
Anyways, the PHP Version requirement of doctrine/collections is not relevant here.